extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -356,33 +356,33 @@ static function isBlocked($text, $blockData, $writeStats = true) {
 				);
 			}
 			wfRestoreWarnings();
-			if ($matched) {
-				if ($writeStats) {
-					self::addStats($blockData['id'], $blockData['type']);
+			if ( $matched ) {
+				if ( $writeStats ) {
+					self::addStats( $blockData['id'], $blockData['type'] );
 				}
 				$result['blocked'] = true;
 				$result['msg'] = $matches[0];
 			}
-		} else { //plain text
-			if (!$blockData['case']) {
-				$text = strtolower($text);
-				$blockText = strtolower($blockText);
+		} else { // plain text
+			if ( !$blockData['case'] ) {
+				$text = strtolower( $text );
+				$blockText = strtolower( $blockText );
 			}
-			if ($blockData['exact']) {
-				if ($text == $blockText) {
-					if ($writeStats) {
-						self::addStats($blockData['id'], $blockData['type']);
+			if ( $blockData['exact'] ) {
+				if ( $text == $blockText ) {
+					if ( $writeStats ) {
+						self::addStats( $blockData['id'], $blockData['type'] );
 					}
 					$result['blocked'] = true;
-					$result['msg'] = $blockData['text'];    //original case
+					$result['msg'] = $blockData['text'];    // original case
 				}
 			} else {
-				if ( !empty($blockText) && strpos($text, $blockText) !== false) {
-					if ($writeStats) {
-						self::addStats($blockData['id'], $blockData['type']);
+				if ( !empty( $blockText ) && strpos( $text, $blockText ) !== false ) {
+					if ( $writeStats ) {
+						self::addStats( $blockData['id'], $blockData['type'] );
 					}
 					$result['blocked'] = true;
-					$result['msg'] = $blockData['text'];    //original case
+					$result['msg'] = $blockData['text'];    // original case
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)